controllers/spiaccesstokenbinding_controller.go

```diff
@@ -83,15 +85,24 @@ func (r *SPIAccessTokenBindingReconciler) SetupWithManager(mgr ctrl.Manager) err
 		For(&api.SPIAccessTokenBinding{}).
 		Owns(&corev1.Secret{}).
 		Watches(&source.Kind{Type: &api.SPIAccessToken{}}, handler.EnqueueRequestsFromMapFunc(func(o client.Object) []reconcile.Request {
+			kcpWorkspace := logicalcluster.From(o)
```

Comment:
Did that compatible with non-KCP env?

Reply:
yes, it will be empty and later set `reconcile.Request{ ClusterName: "" }`, which is same as not set it at all